libs/src/sdk/api/tracks/constants.ts

```diff
@@ -0,0 +1,16 @@
+export const TRACK_REQUIRED_PROPERTIES = [
+  'cover_art_sizes',
+  'credits_splits',
+  'file_type',
+  'genre',
+  'is_premium',
+  'length',
+  'mood',
+  'owner_id',
+  'premium_conditions',
+  'release_date',
+  'tags',
+  'title'
```

Comment:
how about: `is_hidden`/`is_available`, `isrc`, `iswc`, `premium_content_signature`, and `stem_of`? (from the Track object type)

Reply:
Don't think it's necessary to check for any optional props bc entity manager already falls back to default values

Reply:
makes sense